internal/ui/logo_test.go

```diff
@@ -24,17 +24,17 @@ func TestLogoStatus(t *testing.T) {
 		"info": {
 			"[#008000::b] ____  __.________       \n[#008000::b]|    |/ _/   __   \\______\n[#008000::b]|      < \\____    /  ___/\n[#008000::b]|    |  \\   /    /\\___ \\ \n[#008000::b]|____|__ \\ /____//____  >\n[#008000::b]        \\/            \\/ \n",
 			"blee",
-			"[white::b]blee\n",
+			"#ffffffblee\n",
```

Comment:
The original text should not need to be changed. The implementation here is incorrect.

Reply:
Thanks for the review!